bundle/Controller/UserController.php

```diff
@@ -155,20 +155,21 @@ public function editAction(
         ?string $language = null,
         Request $request
     ) {
+        $languageCode = $language;
         $user = $this->userService->loadUser($contentId);
         if (!$this->permissionResolver->canUser('content', 'edit', $user)) {
             throw new CoreUnauthorizedException('content', 'edit', ['userId' => $contentId]);
         }
         $contentType = $this->contentTypeService->loadContentType($user->contentInfo->contentTypeId);
-
+        $language = $this->languageService->loadLanguage($languageCode ?:$this->languageService->getDefaultLanguageCode());
```

Comment:
Are you sure about `?: $this->languageService->getDefaultLanguageCode()` ?

Reply:
I changed these parameters to mandatory after talked Maciej, then LanguageCode always must exist and the user must have permission